WooCommerce/Classes/POS/Presentation/Card Present Payments/Connection Alerts/PointOfSaleCardPresentPaymentFoundReaderAlertViewModel.swift

```diff
@@ -24,11 +25,18 @@ struct PointOfSaleCardPresentPaymentFoundReaderAlertViewModel: Hashable {
 private extension PointOfSaleCardPresentPaymentFoundReaderAlertViewModel {
     enum Localization {
         static let title = NSLocalizedString(
-            "pointOfSale.cardPresentPayment.alert.foundReader.title",
-            value: "Do you want to connect to reader %1$@?",
+            "pointOfSale.cardPresentPayment.alert.foundReader.title.2",
```

Comment:
Ah! Fun times with keys. Thanks for the heads up on p1724920761160859/1724920521.169959-slack-CGPNUU63E